### PR TITLE
MS. Add. A. 263: Peter Kidd description

### PR DESCRIPTION
**Pull request by holfordm**

### CODE REVIEW

File: collections/Add_A/MS_Add_A_263.xml [hunk 1]
```diff
@@ -37,13 +41,39 @@
...
+                  </objectDesc>
+                  <bindingDesc>
+                     <binding contemporary="true">
+                        <p>Contemporary binding. Sewn on four pairs of twisted parchment bands, laced and pegged separately into horizontal channels in wood (beech?) boards, covered with undecorated tawed(?) skin, overlapping the edges of the leaves; two strap-and-pin clasps (closing from the back to the front board); the front cover inscribed with a shelfmark in bold 15th-century characters ‘Y.22.’; the top of the spine inscribed ‘Legen<lb/>da S. | Thom<lb/>as [ … ] | &amp; | S. Aug. | S. Ambr.’ (worn; readings uncertain; cf. fols. 49–54).
+                        </p>
+                     </binding>
+                  </bindingDesc>
                </physDesc>
+               <history>
+                  <provenance>Doubtless written for a Dominican house in Germany, and perhaps still there in the 15th century.</provenance>
```
Comment (adunning) [inline collections/Add_A/MS_Add_A_263.xml:64]: 'Doubtless' seems here and below to be indicating a measure of uncertainty, which should be made clearer.

File: collections/Add_A/MS_Add_A_263.xml [hunk 2]
```diff
@@ -37,13 +41,39 @@
                      <idno type="SCN">29383</idno>
                   </altIdentifier>
                </msIdentifier>
+               <head>Guillelmus de Tocco, <title>Ystoria sancti Thome de Aquino</title>, with offices for Thomas Aquinas and Corpus Christi. A, B Germany, 14th century (after 1318)</head>
```
Comment (adunning) [inline collections/Add_A/MS_Add_A_263.xml:44]: 'A, B Germany' looks like a typo in isolation.

File: collections/Add_A/MS_Add_A_263.xml [hunk 3]
```diff
@@ -37,13 +41,39 @@
...
+                           <dimensions type="leaf" unit="mm">
+                              <height min="240" max="245">240–5</height>
+                              <width max="175" min="170">170–5</width>
+                           </dimensions></extent>
+                        <foliation> 1–64 in late 19th-century purple pencil, i–iii and 65v in 20th-century pencil</foliation>
+                     </supportDesc>
+                  </objectDesc>
+                  <bindingDesc>
+                     <binding contemporary="true">
+                        <p>Contemporary binding. Sewn on four pairs of twisted parchment bands, laced and pegged separately into horizontal channels in wood (beech?) boards, covered with undecorated tawed(?) skin, overlapping the edges of the leaves; two strap-and-pin clasps (closing from the back to the front board); the front cover inscribed with a shelfmark in bold 15th-century characters ‘Y.22.’; the top of the spine inscribed ‘Legen<lb/>da S. | Thom<lb/>as [ … ] | &amp; | S. Aug. | S. Ambr.’ (worn; readings uncertain; cf. fols. 49–54).
```
Comment (adunning) [inline collections/Add_A/MS_Add_A_263.xml:58]: I'm not quite clear on the difference here between `<lb/>` and `|` – doesn't the latter also signify a line break?

File: collections/Add_A/MS_Add_A_263.xml [hunk 4]
```diff
@@ -37,13 +41,39 @@
...
+                  <provenance>Unidentified 19th(?)-century collection, doubtless German: inscribed in ink <q type="pressmark">N XXIV</q> and <q type="pressmark">N 24</q> (fol. ii r).</provenance>
+                  <provenance>Unidentified 19th-century German-speaking owner/bookseller: with an inserted slip inscribed in pencil ‘Beide Theile dieser Codex 14 saec.’ (attached to fol. ii r); his(?) number ‘173’ inside the back board and on a slip of paper attached to the back board.</provenance>
+                  <provenance>H. Grevel &amp; Co., London bookseller: their(?) inventory number ‘9157’ in pencil (fol. iir upper right; fol. 1r, top left; fol. 64r, lower left); their Catalogue 4, <title>Catalogue of rare manuscripts and early printed books …</title> (1884), item 6; pencil price ‘£4’ (fol. iir):</provenance>
+                  <acquisition when="1884">Purchased by the Bodleian on 24 March 1884, for £3 10s; inscribed in pencil ‘Grevel 1884’ (fol. iir). Former Bodleian shelfmarks: ‘MS. Bodl. Adds. A. 263’ (fol. 1r, lower margin; cancelled by encircling).
```
Comment (adunning) [inline collections/Add_A/MS_Add_A_263.xml:70]: Noting 'Adds. A. 263' as a former shelfmark seems pedantic to me, but is there significance that I am missing?

Comment (holfordm) [reply collections/Add_A/MS_Add_A_263.xml:70]: agreed but see comment under Canon Pat Lat 113

File: collections/Add_A/MS_Add_A_263.xml (hunk 3 above)
Comment (holfordm) [reply collections/Add_A/MS_Add_A_263.xml:58]: No difference except that <lb/> is better for searchability, so I tend to use it within words as here. I suppose I should be consistent!

**Pull request merged**